README.md

```diff
@@ -73,4 +73,19 @@ the script `copy-resources.sh` will be run automatically and will copy across th
 
 The file `schema.json` is obligatory, and for PDF validation valid `stylesheet.xsl` and `translations.json` files.
 
-Rendering text fixtures should be added using the same file structure in the `fixtures` folder. Any `valid_certificate_*.json` files will be rendered and the result checked against the corresponding `valid_certificate_*.pdf` file.
\ No newline at end of file
+Rendering text fixtures should be added using the same file structure in the `fixtures` folder. Any `valid_certificate_*.json` files will be rendered and the result checked against the corresponding `valid_certificate_*.pdf` file.
+
+## Converting Schemas from .JSON to .XML
+
+To generate the XML schema from a JSON schema, run the following command:
+
+```shell
+npm run json2xml <relative filepath to schema>
+```
+
+**Example:**
+```shell
+npm run json2xml schemas/CoA/v1.1.0/schema.json
+```
+
+It will save the resulting file to the file system.
```

Comment:
```suggestion
It will save the resulting file to the folder containing the `schema.json` file as a `schema.xml` file.
```

Reply:
I ran the script after writing this and noticed that it saves the xml file to the project root rather than alongside the input file, can you update it to update the xml file in the same folder as the input path? Otherwise it could be confusing for the user. 

You could probably use something like `jsonFilePath.replace('json', 'xml')` as the output path.